src/main/resources/com/google/api/codegen/java/test.snip

```diff
@@ -272,9 +272,15 @@
 
   @join assert : test.asserts
     @if assert.hasExpectedValueTransformFunction
-      Assert.assertEquals({@assert.expectedValueTransformFunction}(\
-        {@assert.expectedValueIdentifier}), \
-        actualRequest.{@assert.actualValueGetter}());
+      @if assert.hasActualValueTransformFunction
```

Comment:
I was imagining a comment in the snippet file (comments are supported, see https://github.com/googleapis/toolkit/blob/master/src/main/resources/com/google/api/codegen/java/main.snip#L275) but outputting a non-compiling string seems better.